test-wasms/invoker_account_exists/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#![no_std]
+use soroban_sdk::{contracterror, contractimpl, panic_error, Address, Env};
+
+#[contracterror]
+#[derive(Copy, Clone)]
+#[repr(u32)]
+pub enum Error {
+    InvokerIsContract = 1,
+}
+
+pub struct Contract;
+
+#[contractimpl]
+impl Contract {
+    pub fn invkexists(env: Env) -> bool {
+        match env.invoker() {
+            Address::Account(account_id) => env.accounts().get(&account_id).is_some(),
+            Address::Contract(_) => panic_error!(&env, Error::InvokerIsContract),
+        }
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use soroban_sdk::{testutils::Accounts, Env};
+
+    use crate::{Contract, ContractClient};
+
+    #[test]
+    fn test_invoker() {
```

Comment:
does this test execute the [updates on invoke path](https://github.com/stellar/soroban-cli/pull/219/files#diff-7933b1f789a38779ee96fb34ae9b41170279f5a2bc35af08fb399fe0e93af89cR380), I'm still learning on rust..

can the test exercise both outcomes, i.e. verify source account doesn't exist and if worthwhile, test the baseline, source account did exist also?